ide/languages.yaml/src/org/netbeans/modules/languages/yaml/YamlKeystrokeHandler.java

```diff
@@ -289,9 +290,9 @@ public int beforeBreak(Document document, int offset, JTextComponent target) thr
         int indent = getLineIndent(doc, offset);
         String linePrefix = doc.getText(lineBegin, offset - lineBegin);
         String lineSuffix = doc.getText(offset, lineEnd + 1 - offset);
-        if (linePrefix.trim().endsWith(":") && lineSuffix.trim().length() == 0) {
+        if (linePrefix.stripTrailing().endsWith(":") && lineSuffix.isBlank()) {
```

Comment:
gonna steal this and put this into a `stripLeading()`/`stripTrailing()` rule -> https://github.com/mbien/jackpot-inspections/commit/037b5867190b49030be754df8c18c9a6646df250 :)